src/controllers/update/update.review.beneficial.owner.individual.ts

```diff
@@ -19,13 +26,26 @@ export const get = (req: Request, res: Response) => {
   if (appData?.beneficial_owners_individual){
     dataToReview = appData?.beneficial_owners_individual[Number(index)];
   }
+
   const backLinkUrl = getBackLinkUrl(appData, index);
-  return res.render(UPDATE_REVIEW_BENEFICIAL_OWNER_INDIVIDUAL_PAGE, {
-    backLinkUrl,
-    templateName: UPDATE_REVIEW_BENEFICIAL_OWNER_INDIVIDUAL_PAGE,
-    ...dataToReview,
-    isOwnersReview: true
-  });
+
+  // Check if ceased_date field exists to populate field for change link after review
```

Comment:
"Ceased date is undefined for initial review of BO - don't set ceased date data in this scenario"